src/components/CldPlaceholder/CldPlaceholder.md

```diff
@@ -0,0 +1,46 @@
+### Attributes
+
+Refer to the base [Cloudinary JS SDK](https://github.com/cloudinary/cloudinary_js#configuration) for configuration options.
+
+See the [Image transformation reference](https://cloudinary.com/documentation/image_transformation_reference) documentation for all the options accepted by the `CldImage` and `CldTransformation` components.
+
+
+### Usage
+
+```jsx
+<cld-image cloudName="demo" publicId="small_dinosaur">
+  <cld-placeholder type="predominant-color"></cld-placeholder>
+</cld-image>
+```
+
+### Lazy mode
```

Comment:
Does that supposed to be in the placeholder component?

Reply:
I think so, the two features are a bit related.

Either we show lazyload here, or show placeholder in the lazyload (to show how they're related).